test/runnable/eh.d

```diff
@@ -755,6 +755,73 @@ void test12989()
 
 /****************************************************/
 
+int bar10(int c)
+{
+    if (c <= 0xFFFF)
+    {
+    L3:
```

Comment:
Is the position of this label important? Otherwise you could put it together with the goto: `L3: goto L3;`